### ISSUE TITLE
fails on last item

### ISSUE THREAD

**Issue by srl295** [pard68/planwarrior#1]
instead of printing the last row it prints

```
  File "planwarrior/__main__.py", line 33, in <module>
    main()
  File "planwarrior/__main__.py", line 29, in main
    print('\n'.join(formatter(p, c, n, parsed_plan[c])))
  File "planwarrior/formatting.py", line 40, in formatter
    if now >= ms and now < next_ms:
TypeError: '<' not supported between instances of 'int' and 'NoneType'
```

… i'm guessing `next_ms` is None when there's no next …
adding a line to the end of the plan causes the prior line to work but the new last line to fail.

**Comment by ispringle** [comment]
I actually just fixed this in b40748333e81f435cda74b057bad5246a7b6a7cf

Mind testing that make sure it's working for you before closing this?

**Comment by srl295** [comment]
Fixed in b407483
